config/read.go

```diff
@@ -405,7 +405,7 @@ func CreateHTTPClient(conf ServerConfig, logger *util.Logger, retry bool) *http.
 		client := retryablehttp.NewClient()
 		client.RetryWaitMin = 1 * time.Second
 		client.RetryWaitMax = 30 * time.Second
-		client.RetryMax = 4
+		client.RetryMax = 2
```

Comment:
Can we map out the effective difference of how long we waited at most before in total, vs how long we wait now? (based on https://github.com/hashicorp/go-retryablehttp/blob/v0.7.0/client.go#L470)

Mainly I'm wondering if there is an argument that its good to wait a few minutes if our backend services are temporarily unavailable (if the max was always used, previously we waited at most 2 mins, and now we'd wait at most 1 minute -- but I think in practice those numbers would be lower, given the exponential backoff logic that's capped to that max)

Reply:
As I remember it, HTTP retry was added to work around temporary network connectivity issues, not to smooth over API outages (which should be very rare).